package/libwebsockets/libwebsockets.mk

```diff
@@ -15,9 +15,6 @@ LIBWEBSOCKETS_CONF_OPTS = \
 	-DLWS_IPV6=ON \
 	-DLWS_UNIX_SOCK=ON \
 	-DLWS_WITHOUT_EXTENSIONS=OFF
-LIBWEBSOCKETS_CONF_OPTS += \
-	-DLWS_WITH_EXTERNAL_POLL=ON \
-	-DDISABLE_WERROR=ON
```

Comment:
If we remove this, the build will fail.

Reply:
I just tried a `make falcon-ucb-libwebsockets-rebuild` and didn't see any issues. What setup were you using when it failed?

Reply:
Was a previous master build, but maybe things have changed since then.

Reply:
I thought the failure was in linking against it from emb-core? but, might need to do a full clean `falcon-ucb` to get the whole image there.

Reply:
I guess we'd have to update the emb-platform submodule reference and run a CI build to be 100% sure.

Reply:
Correc, this will need a full build to verify, if I recall correctly.  The other option is that maybe we just fix this issue upstream in libwebsockets with a patch?  The author seems responsive and may merge if it's a simple fix to fix the warning.

Reply:
It's not failing on the native build either now. My guess is turning off external event loop support disabled that code path.

Reply:
I'm doing a full build now to verify this and another issue I might have been seeing.